pkg/ingester/stream.go

```diff
@@ -49,7 +50,8 @@ var (
 )
 
 var (
-	ErrEntriesExist = errors.New("duplicate push - entries already exist")
+	ErrEntriesExist    = errors.New("duplicate push - entries already exist")
+	ErrStreamRateLimit = errors.New("stream rate limit exceeded")
```

Comment:
Sorry I missed this before this was merged, but could you open a separate PR to move this error message to the validate.go file and also update it to include an action for the user to take (similar to other errors in that file).

We should always try to provide an actionable error message for anything returned by the API.